worker/tsconfig.json

```diff
@@ -34,11 +36,21 @@
     "moduleResolution": "node",
     "baseUrl": "./src",
     "paths": {
-      "@core/*": ["core/*"],
-      "@sms/*": ["sms/*"],
-      "@email/*": ["email/*"],
-      "@telegram/*": ["telegram/*"],
-      "@shared/*": ["../../shared/src/*"]
+      "@core/*": [
```

Comment:
is there some ESLint change that resulted in these formatting changes?

Reply:
fixed from code styles